app/config/frontendAppConfig.scala

```diff
@@ -44,6 +44,6 @@ object FrontendAppConfig extends AppConfig with ServicesConfig {
   override lazy val analyticsHost = loadConfig(s"google-analytics.host")
   override lazy val reportAProblemPartialUrl = s"$contactHost/contact/problem_reports_ajax?service=$contactFormServiceIdentifier"
   override lazy val reportAProblemNonJSUrl = s"$contactHost/contact/problem_reports_nonjs?service=$contactFormServiceIdentifier"
-  override lazy val signInUrl = s"$caFrontendHost/gg/sign-in?continue=$apiRevocationFrontendHost/api-revocation/applications"
+  override lazy val signInUrl = s"$caFrontendHost/gg/sign-in?continue=$apiRevocationFrontendHost/applications-permissions-withdrawal/applications"
```

Comment:
Can we use the relative continueUrl? GG teams now supports to have relative continueUrls.
There is blog post on conflueance about this.